frontend/packages/integration-tests-cypress/tests/cluster-settings/alertmanager/alertmanager.cy.ts

```diff
@@ -74,11 +74,11 @@ describe('Alertmanager', () => {
     cy.byTestID('webhook-url').type(webhookURL);
     cy.byTestID('label-0').type(label);
     alertmanager.save();
-    alertmanager.validateCreation(receiverName, receiverType, label);
-    listPage.rows.clickKebabAction(receiverName, 'Delete Receiver');
+    alertmanager.validateCreation(receiverName, 'integration-types', 'routing-labels');
```

Comment:
`receiverType` no longer needs to be a const.

Reply:
Constants are for resharing.  Since `receiverName` is only used once, there is no need for it to exist.